libs/docs/src/navigation/2.2.0-alpha.yaml

```diff
@@ -12,6 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- title: Get started
+  links:
+    - title: "Hello world example: odo v3"
+      href: ./helloworld-example-odo
+    - title: Developing with devfiles
+      href: ./developing-with-devfiles
+    - title: Integrate with editors
+      href: ./integrate-with-editors
+    - title: Library
+      href: ./library
```

Comment:
I think we can remove `2.2.0-alpha` since we are on `2.2.1-alpha` now

Reply:
This is now being taken care of in PR https://github.com/devfile/devfile-web/pull/89.